### PR TITLE
Backend support for textual inversions (embeddings)

### PR DESCRIPTION
**Pull request by JeLuF**
Basic backend support for embeddings. It's missing some comfort features (e.g. feedback which embeddings couldn't be loaded), but is in a usable state.

### CODE REVIEW

**Comment by JeLuF** [comment]
Added compel TI support. Thanks rbertus2000!

**Review by rbertus2000 — APPROVED** [review]
looks good to me!

File: setup.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
         "safetensors",
         "k-diffusion",
         "diffusers==0.15.1",
```
Comment (cmdr2) [inline setup.py:13]: Making a note to myself to upgrade diffusers after merging this, and test before pushing to beta

**Comment by cmdr2** [comment]
Thanks!

**Pull request merged**